AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/InitialSyncOrchestrator.swift

```diff
@@ -65,37 +67,33 @@ final class AWSInitialSyncOrchestrator: InitialSyncOrchestrator {
         syncOperationQueue.isSuspended = false
     }
 
-    /// Creates a graph of registered models and returns the roots.
-    private func getModelRoots() -> [DirectedGraphNode<Model.Type>] {
-        let syncableModels = ModelRegistry.models.filter { !$0.schema.isSystem }
-        let modelGraphs = ModelGraphs(models: syncableModels)
-        let roots = modelGraphs.roots
-        return roots
+    private func enqueueSyncableModels() {
+        let syncableModels = ModelRegistry.models.filter { $0.schema.isSyncable }
```

Comment:
We did get rid of the isSyncable attribute; I didn't completely rip out the attribute because it's referenced in a number of places (I thik in the API category as well). I'll look at removing that in an upcoming PR